cmscontrib/loaders/simple_csv.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+
+# Programming contest management system
+# Copyright © 2016 Karlis Senko <karlis3p70l1ij@gmail.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import os
+from backports import csv  # Use native csv reader in python3
+import io
+
+from cms.db import User
+
+from .base_loader import UserLoader
+
+logger = logging.getLogger(__name__)
+
+
+class CsvUserLoader(UserLoader):
+    """Load users from CSV file.
+
+    Flexible CSV user loader.
+
+    """
+
+    def get_task_loader(self, taskname):
+        raise Exception("get_task_loader is irrelevant for user loader")
```

Comment:
It makes it clear for static analysis that this is intentional and prevents it  from complaining about unimplemented abstract methods.